nipype/interfaces/mrtrix3/utils.py

```diff
@@ -822,7 +823,7 @@ class MRTransformInputSpec(MRTrix3BaseInputSpec):
     )
     invert = traits.Bool(
         argstr="-inverse",
-        position=1,
+        position=2,
```

Comment:
This means `-invert` will appear after all of the things with `position=1`. Does it need to come last, before in_files/out_file?

Reply:
I removed the position change for now, but I just added this previously because if I wanted to use -inverse with some other parameter and they both had 'position=1' then only one would show up on the command. But it can also be fixed by just using inputs.args string.